tests/nlu/selectors/test_selectors.py

```diff
@@ -282,3 +285,54 @@ async def test_train_persist_load(component_builder: ComponentBuilder, tmpdir: P
     await _train_persist_load_with_different_settings(
         pipeline, component_builder, tmpdir, True
     )
+
+
+async def test_process_gives_diagnostic_data(
+    component_builder: ComponentBuilder, tmp_path: Path
```

Comment:
can we use an existing fixture with a trained model instead?

Reply:
I didn't see any fixture that trains a response selector, but I'll look again.

Reply:
I found this one https://github.com/RasaHQ/rasa/blob/bb609de9e0e4912619929cfbef7d06ed9ef045a7/tests/core/conftest.py#L222

Reply:
that worked